moduleframework/mtf_scheduler.py

```diff
@@ -229,21 +248,40 @@ def __init__(self, args, unknown):
         core.print_debug("additionalAvocadoArg = {0}".format(
             self.additionalAvocadoArg))
 
+    def check_tests(self):
+        output = subprocess.check_output([self.AVOCADO, "list", "-V", "--"] + self.tests)
+        assert "TEST TYPES SUMMARY" in output
+        badstates = ["NOT_A_TEST", "MISSING", "ACCESS_DENIED", "BROKEN_SYMLINK"]
+        badtests = []
+        # remove fist line, it is header, and remove last 11 lines, what is test summary
+        for testline in output.split("\n")[1:-11]:
```

Comment:
avocado does not have something like `--no-header`. This looks like to me as RFE to avocado. What are last 11 lines?

Reply:
this is avocado output (11 lines are this summary, it is mentioned  in comment "and remove last 11 lines, what is test summary")
```
$ avocado list -V a b c
Type       Test                                                  Tag(s)
NOT_A_TEST a: File not found ('a'; '/usr/share/avocado/tests/a') 
NOT_A_TEST b: File not found ('b'; '/usr/share/avocado/tests/b') 
NOT_A_TEST c: File not found ('c'; '/usr/share/avocado/tests/c') 

TEST TYPES SUMMARY
==================
ACCESS_DENIED: 0
BROKEN_SYMLINK: 0
EXTERNAL: 0
INSTRUMENTED: 0
MISSING: 0
NOT_A_TEST: 3
PYUNITTEST: 0
SIMPLE: 0
```

Reply:
And what if output is changed? We have to think about it for long time. I would prefer to discuss it with avocado team to provide computer readable format. E.g. JSON or YAML.